sdk/resourcemanager/azure-resourcemanager-redis/src/main/java/com/azure/resourcemanager/redis/implementation/ConfigurationUtils.java

```diff
@@ -24,6 +24,12 @@ static Map<String, String> toMap(RedisConfiguration configuration) {
             if (configuration.maxmemoryPolicy() != null) {
                 map.put("maxmemory-policy", configuration.maxmemoryPolicy());
             }
+            if (configuration.storageSubscriptionId() != null) {
+                map.put("storage-subscription-id", configuration.storageSubscriptionId());
+            }
+            if (configuration.aadEnabled() != null) {
+                map.put("aad-enabled", configuration.aadEnabled());
+            }
```

Comment:
These changes are probably due to this test not executed during last api-version update.